server/src/main/scala/org/elasticmq/server/ElasticMQServer.scala

```diff
@@ -65,6 +68,28 @@ class ElasticMQServer(config: ElasticMQServerConfig) extends Logging {
     }
   }
 
+  private def optionallyStartRestStatistics(queueManagerActor: ActorRef): Option[StatisticsRestServer] = {
+    if (config.restStatisticsConfiguration.enabled) {
+
+      val server = TheStatisticsRestServerBuilder(
+        Some(actorSystem),
+        Some(queueManagerActor),
```

Comment:
since these are always `Some` we can probably simplify by using non-optional parameters here (I know it's copied from the sqs server, but the optionals over there are a historical artifact, probably can also be removed, but that's another story :) )

Reply:
Sure, I'll take care of that ;)

Reply:
I've refactored TheStatisticsRestServerBuilder to not use optional actor system. I've looked also on SQSRestServerBuilder and have some doubts about changing that one. I could force to use non-optional value, but It could break some backward compatibility if anyone used it already. We are using that's None possibility on some tests.
@adamw do you think that changing only Statistics builder will be enough?

Reply:
Yes, let's leave `SQSRestServerBuilder` for another occasion. That's not the scope of this task, better to keep PRs small :)

Reply:
👍